man/man1/pmseries.1

```diff
@@ -721,6 +721,13 @@ then a short-hand form can be used:
 .SAMPLE
 $ pmseries --load $PCP_LOG_DIR/pmlogger/acme.0
 .ESAMPLE
+.PP
+NOTE: Timeseries loading is
+.B append-only
+(timestamp-wise) and if more than
+.B stream.maxlen
+entries (defined in /etc/pcp/pmproxy/pmproxy.conf) are loaded for
```

Comment:
This reference to /etc/pcp needs to be via PCP_SYSCONF_DIR as it may differ between platforms - I'll fix that up.